ansible/site.yaml

```diff
@@ -0,0 +1,8 @@
+- name: Install Jenkins
+  hosts: jenkins
+  become: yes
+  become_method: sudo
```

Comment:
Not needed. 
https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-become-method